swagger.json

```diff
@@ -729,10 +750,19 @@
             },
             "webhooks" : {
               "type" : "boolean"
+            },
+            "defaultReviewers": {
```

Comment:
What do you think, worth that we add a description to the fields like we do in the UI, e.g.

"Requires Repository permissions synchronization."

Reply:
Ah yes, that's probably a good think to do...